torch/utils/_python_dispatch.py

```diff
@@ -1,34 +1,156 @@
-import torch
 import contextlib
 from typing import Iterator
+import functools
+
+from torch.utils._mode_utils import _enable_mode, _push_mode, _ModeInfo, _wrap_init
+from torch._C import _get_python_mode, _set_python_mode
+from dataclasses import dataclass
+
+
+@dataclass
+class PythonModeInfo(_ModeInfo):
```

Comment:
yeah, my vote is for just fixing the naming all in one go in this PR

Reply:
for ease in migration, you can keep `enable_python_mode` BC stub, but everything else should be in the new name. We should also decide what public module we're going to put this in.